tests/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace RevAPI;
+
+$autoloader = require __DIR__ . '/../vendor/autoload.php';
+$autoloader->add(__NAMESPACE__, __DIR__);
+
+if (file_exists(__DIR__ . '/config.inc.php')) {
+    require_once __DIR__ . '/config.inc.php';
```

Comment:
I realize this is a test bootstrap so it's not a huge stink. However, good coding practice suggests that we should avoid loading configuration files that could have side-effects. That is, if I included or required the config file again bad things wouldn't happen. It's the bootstrap's responsibility for mapping static configuration files into the environment.

A better setup would be to have `config.inc.php` contain:
```php
<?php
return [
    'CLIENT_KEY' => 'yourkey',
    'USER_KEY' => 'yourotherkey',
];
```
Then have the bootstrap load that data into the environment:
```php
$config = include __DIR__ . '/config.inc.php';
$_ENV['REV_CLIENT_API_KEY'] = $config['CLIENT_KEY'];
// etc
```
There are no side-effects from returning an array in an include. :)